support/util/util.go

```diff
@@ -141,3 +143,20 @@ func decompress(r io.Reader) (*bytes.Buffer, error) {
 
 	return bytes.NewBuffer(data), nil
 }
+
+func IsHostnameReachable(hostname string) bool {
+	port := ""
+	if strings.Contains(hostname, ":") {
+		address := strings.Split(hostname, ":")
+		hostname = address[0]
+		port = address[1]
+	}
+
+	timeout := 1 * time.Second
```

Comment:
I'm thinking another alternative approach would be to validate by trying to use the user admin kubeconfig to build a client successfully. Thoughts cc @csrwng?

Reply:
I think that would be better :+1: